src/main/java/uk/ac/ebi/ena/webin/cli/utils/UrlUtils.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright 2018-2021 EMBL - European Bioinformatics Institute
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this
+ * file except in compliance with the License. You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ * CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package uk.ac.ebi.ena.webin.cli.utils;
+
+public class UrlUtils {
+    public final static String WEBIN_REST_V1_TEST_URL = "https://wwwdev.ebi.ac.uk/ena/submit/drop-box/";
```

Comment:
Optimally the WebinService in webin-cli-validation would also use the WebinSubmissionRestV1/V2 naming convention:

```
  final String resolveAgainstWebinSubmissionRestV1Url(String uri) {
        return webinSubmissionRestV1Uri + uri;
    }

    final String resolveAgainstWebinSubmissionRestV2Uri(String uri) {
        return webinSubmissionRestV1Uri + uri;
    }
```